extensions/wikia/WikiaApi/WikiaApiQueryAllUsers.php

```diff
@@ -149,15 +137,15 @@ private function global_users() {
 		$db = $this->getSharedDB();
 
 		$limit = $this->params['limit'];
-		$this->addTables('`user`', 'u1');
+		$this->addTables( '`user`', 'u1' );
 
 		if ( !is_null( $this->params['group'] ) ) {
 			$users = $this->getUsersForGroup();
-			if ( empty($users) ) 
-				$users = array(0);
+			if ( empty( $users ) )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add curly braces around the nested statement(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)